libs/ngx-cron-editor/src/cron-editor.component.ts

```diff
@@ -51,8 +51,6 @@ function* range(start: number, end: number) {
   providers: [CRON_VALUE_ACCESSOR]
 })
 export class CronGenComponent implements OnInit, OnDestroy, ControlValueAccessor {
-  public tabIndex = 0;
```

Comment:
I was sure we needed tabIndex to show the correct tab based on the input expression. Can you verify that this works?

Reply:
I will verify

Reply:
You are right, it's needed to show the correct tab based on the input expression. I didn't notice this before because I had no bound input. I'll update PR.

Reply:
Thanks! I will merge the pull request shortly.